linodecli/api_request.py

```diff
@@ -150,6 +150,19 @@ def _build_filter_header(
         if p.name in parsed_args_dict:
             del parsed_args_dict[p.name]
 
+    # check for order_by and order
+    order_by, order = None, None
+    if parsed_args_dict["order_by"] is not None:
+        order_by = parsed_args_dict["order_by"]
+        order = (
+            "asc"
+            if parsed_args_dict["order"] is None
+            else parsed_args_dict["order"]
+        )
+
+    del parsed_args_dict["order_by"]
+    del parsed_args_dict["order"]
+
```

Comment:
This logic can be simplified down a bit:

```suggestion
    # check for order_by and order
    order_by = parsed_args_dict.pop("order_by")
    order = parsed_args_dict.pop("order") or "asc"
```
See below for the corresponding suggestion.

Reply:
Will definitely do that, that is so much cleaner!